dex/path.go

```diff
@@ -5,62 +5,48 @@ package dex
 
 import (
 	"os"
-	"os/user"
 	"path/filepath"
-	"runtime"
 	"strings"
 )
 
-// CleanAndExpandPath expands environment variables and leading ~ in the
-// passed path, cleans the result, and returns it.
+// CleanAndExpandPath expands environment variables and leading ~ in the passed
+// path, cleans the result, and returns it.
 func CleanAndExpandPath(path string) string {
 	// Nothing to do when no path is given.
 	if path == "" {
 		return path
 	}
 
-	// NOTE: The os.ExpandEnv doesn't work with Windows cmd.exe-style
-	// %VARIABLE%, but the variables can still be expanded via POSIX-style
-	// $VARIABLE.
-	path = os.ExpandEnv(path)
+	dirName := ""
+
+	// This supports Windows cmd.exe-style %VARIABLE%.
+	if strings.HasPrefix(path, "%") {
+		// Split path into %VARIABLE% and path
+		pathArray := strings.SplitAfterN(path, "/", 2)
+		dirEnv := strings.ToUpper(strings.Trim(pathArray[0], "%/"))
+		path = pathArray[1]
```

Comment:
@chappjc thanks for your feedback, will refactor this. 
> Path to application directory. The default is ~/.dexc on UNIX OSes,
~/Library/Application Support/Dexc on macOS and %localappdata%\Dexc on
Windows.

Thought of support for windows variables with % since config files contain reference  to `%localappdata%\Dexc`, plus windows users might not naturally use `~/path` to expand to user home directory.

Reply:
I really can't comment on what's best for windows cli users... powershell is very similar to bash and does understand and expand `~`